include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -86,6 +86,23 @@ namespace internal
         result[component] = vector_entry;
       }
 
+      static scalar_value_type
+      sum_value(const scalar_value_type &result)
```

Comment:
@bergbauer I was not sure about the traits. Is there an easier way to accomplish the same?